ecs_fargate/README.md

```diff
@@ -132,9 +157,11 @@ The ECS Fargate check does not include any events.
 
 ### Service Checks
 
-**fargate_check**  
```

Comment:
This double space is needed. Markdown reads double-spaces at the end of a line as a `<br>` or line-down.

Reply:
👍